lib/tasks/generate_key.rake

```diff
@@ -0,0 +1,8 @@
+namespace :auth do
+  desc 'run locally to generate a key for use as the shared secret'
+  task :generate_key do
+    puts 'Use this key by setting it as a the value for DEVELOPMENT_JWE_SHARED_KEY or JWE_SHARED_KEY env vars'
```

Comment:
I would also suggest the same as Phil and go for `DEVELOPMENT_JWE_SHARED_KEY` or `JWE_SHARED_KEY`. Its what everyone is used to and is simpler.